.env

```
@@ -0,0 +1,9 @@
+# Database Configuration
+POSTGRES_DB=SHOPPMATE
+POSTGRES_USER=user
+POSTGRES_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)